.github/workflows/build.yml

```diff
@@ -196,4 +194,25 @@ jobs:
           tags: |
             "${{ secrets.GCP_AR_PARABOL_DEV }}:${{github.sha}}"
             "${{ env.DOCKER_REPOSITORY_FOR_REF }}:${{ env.GITHUB_REF_NAME_NORMALIZED }}"
-          # platforms: linux/amd64,linux/arm64 # Uncomment to be able to build multi-platform images.
+      - name: Push Artifacts to Sentry
+        if: github.ref_type == 'tag'
+        uses: getsentry/action-release@v1
+        env:
+          SENTRY_AUTH_TOKEN: "${{secrets.SENTRY_AUTH_TOKEN}}"
+          SENTRY_ORG: "parabol"
+          SENTRY_PROJECT: "action-production"
```

Comment:
Shouldn't we upload it to both action-production and action-staging projects?

Reply:
the name is a bit of a misnomer, staging is an environment inside the project `action-production`.

Reply:
🤔 And do we want that? Some time ago I created two new projects in Sentry. Right now we have:
- action-production: used by both Dokku environments (production and staging) in Digital Ocean.
- action-staging: used right now only by the GCP Staging.
- local-development: created in case someone wanted to use sentry in their local environment.
- on-demand-environments: isolated project for the on-demand envs.

I don't really know if that make sense. I created it to isolate the new GCP envs I was building, at least while building and testing.

Reply:
IMO a project refers to a codebase. we have 1 project, which is "action", and that is deployed to many environments. In sentry you can search & filter across envs, but not across projects (at least that was the case, not sure if still true).